src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -1178,21 +1178,21 @@ def _update_private_endpoint_connection_status(cmd, client, resource_group_name,
 
 
 def approve_private_endpoint_connection(cmd, client, resource_group_name, vault_name, private_endpoint_connection_name,
-                                        approval_description=None, connection_id=None):
+                                        description=None, connection_id=None):
     """Approve a private endpoint connection request for a Key Vault."""
 
     return _update_private_endpoint_connection_status(
         cmd, client, resource_group_name, vault_name, private_endpoint_connection_name, is_approved=True,
-        description=approval_description, connection_id=connection_id
+        description=description, connection_id=connection_id
     )
 
 
 def reject_private_endpoint_connection(cmd, client, resource_group_name, vault_name, private_endpoint_connection_name,
-                                       rejection_description=None, connection_id=None):
+                                       description=None, connection_id=None):
     """Reject a private endpoint connection request for a Key Vault."""
 
     return _update_private_endpoint_connection_status(
         cmd, client, resource_group_name, vault_name, private_endpoint_connection_name, is_approved=False,
-        description=rejection_description, connection_id=connection_id
+        description=description, connection_id=connection_id
```

Comment:
actually I think connection_id is unused here, because for client function get() and put() there is no parameter for connection id.

We add such extra parameter to extend customer usage and we should delete it from namespace after parse names related information. 

The same for you _update_xx() function.

Reply:
Good idea.